### ISSUE TITLE
[cob_teleop] recovery of r@w base takes too long

### ISSUE THREAD

**Issue by mgruhler** [4am-robotics/cob_command_tools#131]
Currently, on r@w, the recovery of the base takes very long, approx. 30s.

The problem actually is the new teleop. Even though [this answer](https://github.com/ipa320/cob_command_tools/pull/109#issuecomment-141005264) to [my question](https://github.com/ipa320/cob_command_tools/pull/109#issuecomment-140797161) whether this node works without having the `cob_sound` node started is correct in the sense that it works, there behaviour is not satisfactory. And I don't want to start `cob_sound` on a robot without speakers.

Some (hopefully correct) insights:

The problem is actually that the `say()` function [here](https://github.com/ipa320/cob_command_tools/blob/indigo_dev/cob_teleop/ros/src/cob_teleop.cpp#L357-L366) is called multiple times in the `recover` part of teleop [here](https://github.com/ipa320/cob_command_tools/blob/indigo_dev/cob_teleop/ros/src/cob_teleop.cpp#L404-L442), effectively resulting in multiple `waitForResult(5)` calls if the respective server is not there.

How should this be handled? I`d vote for a parameter to turn of this feature completely!
@ipa-srd @ipa-flg @ipa-nhg @ipa-fxm @ipa-fmw

**Comment by fmessmer** [comment]
Apparently, the current yaml layout does not allow to properly configure the new teleop any hardware....I assume a similar thing happens when using the new teleop on a robot without lights!

A suggestion would be to have four parameters for that:
`enable_light`
`light_action_name`
`enable_sound`
`sound_action_name`

As the sound action goal is sent in a dedicated function `say`, on robots without sound,  we should simply return from this function without sending any goal...i.e.

```
void CobTeleop::say(std::string text, bool blocking)
{
  if(enable_sound_)
  {
    cob_sound::SayGoal saygoal;
    saygoal.text = text;
    sayclient_->sendGoal(saygoal);
    if (blocking)
    {
      sayclient_->waitForResult(ros::Duration(5));
    }
  }
}
```

where `enable_sound_` is a member variable initialized based on  the parameter `enable_sound`

**Comment by mgruhler** [comment]
+1 Actually, I had the same idea for the say function.

Didn't think about the light, but also good to change this!

**Comment by fmessmer** [comment]
solved in #132 
please test
